--- a/src/modules/mpris/mpris.cpp
+++ b/src/modules/mpris/mpris.cpp
@@ -19,7 +19,7 @@
 const std::string DEFAULT_FORMAT = "{player} ({status}): {dynamic}";

 Mpris::Mpris(const std::string& id, const Json::Value& config)
-    : ALabel(config, "mpris", id, DEFAULT_FORMAT, 0, false, true),
+    : ALabel(config, "mpris", id, DEFAULT_FORMAT, 0, true, true),
       tooltip_(DEFAULT_FORMAT),
       artist_len_(-1),
       album_len_(-1),
